extensions/amp-carousel/0.1/slidescroll.js

```diff
@@ -536,4 +541,35 @@ export class AmpSlideScroll extends BaseSlides {
       event.stopPropagation();
     });
   }
+
+  /**
+   * @param {number} newSlideIndex
+   * @private
+   */
+  triggerAnalyticsEvent_(newSlideIndex) {
+    let direction = newSlideIndex - this.slideIndex_;
+    if (direction === 0) {
+      return;
+    } else if (Math.abs(direction) !== 1) {
+      direction = direction < 0 ? 1 : -1;
+    }
+    if (direction === 1) {
+      this.analyticsEvent_('amp-carousel-next');
```

Comment:
Please also add `amp-carousel-change` event (try different names). Most of times, only this event would be interesting, I believe.